### ISSUE TITLE
Add price range multipliers, like for Bookable products

### ISSUE THREAD

**Issue by madeincosmos** [woocommerce/woocommerce-accommodation-bookings#162]
#### Affected ticket(s)

830022-zen

#### What I expected
To have 4 pricing options - weekday low season, weekend low season, weekday high season, weekend high season.

#### What happened instead

This is possible with standard Bookable products - I can add one rule to increase the price on the weekends, and another one to increase the price in high season:

![Screenshot](http://cld.wthms.co/9rOIGg+)
Screenshot: http://cld.wthms.co/9rOIGg

However, on accommodations product I can only set cost per night, without an option to stack rules and increase / multiply the previous price:

![Screenshot](http://cld.wthms.co/9hZ8Sl+)
Screenshot: http://cld.wthms.co/9hZ8Sl

#### Steps to reproduce the issue

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by Luminus** [comment]
The customer on 823246-zen brought this up as well.

**Comment by iMazed** [comment]
Another one in 887476-zen.

**Comment by Drivingralle** [comment]
We have a high interest in enabling shop/booking managers to give discounts to long term bookings.
As fare as I understand this issue this would enable it.

I played around with the rules today and find that they are quite strange in the calculation. If two rules match I'm not always sure witch rule wins.

The reason I was playing with it was that I created a patch to add the option to add rules about "night count discouts":
https://cloudup.com/cK0vwkf_HKl

IMO the current may rules are used to modify the price is not good. 

Happy to work on this in the next days.
